dev_app.ps1

```diff
@@ -21,6 +21,8 @@ function executeScript {
 	iex ((new-object net.webclient).DownloadString("$helperUri/$script"))
 }
 
+executeScript "CompatibilityChecks.ps1";
```

Comment:
what do you think of putting these include statements inside the helper scripts that use them instead of in the recipes?  Since the helper scripts now have a dependency on CompatibilityChecks it would be best to add the include there or the helper scripts become more fragile.

Reply:
Hadn't thought about that (fairly new to PowerShell scripting, more used to Python) but that does make sense